<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KismetEventWindow</class>
 <widget class="QWidget" name="KismetEventWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>762</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>事件库</string>
  </property>
  <property name="windowIcon">
   <iconset resource="orion2.qrc">
    <normaloff>:/images/express.png</normaloff>:/images/express.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>插件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pluginpushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="func">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>301</width>
          <height>691</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>查找</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterlineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="refreshpushButton">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="orion2.qrc">
               <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTabWidget" name="evttabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::North</enum>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="icon">
             <iconset resource="orion2.qrc">
              <normaloff>:/images/condition.png</normaloff>:/images/condition.png</iconset>
            </attribute>
            <attribute name="title">
             <string>条件</string>
            </attribute>
            <widget class="EventTreeWidget" name="cndtreeWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>321</width>
               <height>671</height>
              </rect>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="icon">
             <iconset resource="orion2.qrc">
              <normaloff>:/images/action.png</normaloff>:/images/action.png</iconset>
            </attribute>
            <attribute name="title">
             <string>动作</string>
            </attribute>
            <widget class="EventTreeWidget" name="acttreeWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>321</width>
               <height>671</height>
              </rect>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="icon">
             <iconset resource="orion2.qrc">
              <normaloff>:/images/express.png</normaloff>:/images/express.png</iconset>
            </attribute>
            <attribute name="title">
             <string>表达式</string>
            </attribute>
            <widget class="EventTreeWidget" name="exptreeWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>10</y>
               <width>321</width>
               <height>671</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <stylestrategy>PreferDefault</stylestrategy>
              </font>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="plugin">
       <widget class="OrionTreeWidget" name="plugintreeWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>301</width>
          <height>691</height>
         </rect>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EventTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>eventtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>OrionTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>oriontreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="orion2.qrc"/>
 </resources>
 <connections/>
</ui>
